File: app/src/main/java/com/worldventures/dreamtrips/modules/config/model/VideoRequirement.java
package com.worldventures.dreamtrips.modules.config.model;

public class VideoRequirement {

   public static final int DEFAULT_MAX_VIDEO_DURATION_SEC = 90;

   private int videoMaxLength;

   public int getVideoMaxLength() {
      return videoMaxLength == 0 ? DEFAULT_MAX_VIDEO_DURATION_SEC : videoMaxLength;
   }

   public void setVideoMaxLength(int videoMaxLength) {
      this.videoMaxLength = videoMaxLength;
   }
}

File: app/src/main/java/com/worldventures/dreamtrips/modules/config/model/VideoRequirement.kt
package com.worldventures.dreamtrips.modules.config.model

import com.esotericsoftware.kryo.DefaultSerializer
import com.esotericsoftware.kryo.serializers.CompatibleFieldSerializer

@DefaultSerializer(CompatibleFieldSerializer::class)
data class VideoRequirement(val videoMaxLength: Int = DEFAULT_MAX_VIDEO_DURATION_SEC) {

   companion object {
      val DEFAULT_MAX_VIDEO_DURATION_SEC = 90
   }
}
